mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -112,6 +114,7 @@ CLASS aii net/minecraft/entity/LivingEntity
 	METHOD cx isUndead ()Z
 	METHOD cy getHealth ()F
 	METHOD cz getRecentDamageSource ()Lahi;
+	METHOD d dropAllDrops (Lahi;)V
```

Comment:
Is this dropping with respect to damage source? Needs a better name.

Reply:
IMO the name makes sense in the context of the other methods around it. It's near `dropInventory`, `dropEquipment`, and `dropLoot`, and it calls all three of them, so `dropAllDrops`.

Reply:
Then you should just name it `drop`. You need to be concise.

Reply:
sure